CONTRIBUTING.md

```diff
@@ -89,6 +89,12 @@ and `package-lock.json` files. Run this command to install them in the project d
 
 `npm install`
 
+### Fonts
+
+In order to build the project's [fonts](scripts/FONTS.md), install the `google-font-installer` package into npm with:
+
+`npm install -g google-font-installer`
```

Comment:
Should this be a formal dependency?

Reply:
How do I specify a global dependency formally?

Reply:
Ah ok, I wasn't familiar with `npm exec`